README.md

```diff
@@ -189,21 +205,21 @@ Example:
+   + Manually push the DynamoDB data to Elasticsearch using a Lambda function that you create yourself.
+   + Refresh the DynamoDB table with a backup.
+   + Remove all data from the DynamoDB table, thus creating parity between Elasticsearch and DynamoDB.
 
-+ Support for STU3 and [R4](https://www.hl7.org/fhir/validation.html) releases of FHIR is based on the JSON schema provided by HL7. The schema for R4 is more restrictive than the schema for [STU3](http://hl7.org/fhir/STU3/validation.html). The STU3 schema doesn’t restrict appending additional fields into the POST/PUT requests of a resource, whereas the R4 schema has a strict definition of what is permitted in the request. You can access the schema [here](https://github.com/awslabs/fhir-works-on-aws-routing/blob/mainline/src/router/validation/schemas/fhir.schema.v3.json).
++ Support for STU3 and [R4](https://www.hl7.org/fhir/validation.html) releases of FHIR is based on the JSON schema provided by HL7. The schema for R4 is more restrictive than that for [STU3](http://hl7.org/fhir/STU3/validation.html). The STU3 schema doesn’t restrict appending additional fields into the POST/PUT requests of a resource, whereas the R4 schema has a strict definition of what is permitted in the request. You can access the schema [here](https://github.com/awslabs/fhir-works-on-aws-routing/blob/mainline/src/router/validation/schemas/fhir.schema.v3.json).
 
-**Note**: We are using the official schema provided by [HL7](https://www.hl7.org/fhir/STU3/downloads.html). 
+**Note**: FHIR Works uses the official schema provided by [HL7](https://www.hl7.org/fhir/STU3/downloads.html).  
 
-+ When making a `POST`/`PUT` request to the server, if you get an error that includes the text `Failed to parse request body as JSON resource`, check that you've set the request headers correctly. The header for `Content-Type` should be either `application/json` or `application/fhir+json`. If you're using Postman for making requests, in the **Body** tab, make sure to change the setting to `raw` and `JSON`.
++ When issuing a `POST`/`PUT` request to the server, if you get an error response that includes the text `Failed to parse request body as JSON resource`, check that you've set the request headers correctly. The header for `Content-Type` should be either `application/json` or `application/fhir+json`. If you're using Postman for making requests, in the **Body** tab, make sure to change the setting to `raw` and `JSON`.
 ![Postman Body Request Settings](resources/postman_body_request_settings.png)
 
 ## Feedback
-We'd love to hear from you! Please reach out to our team via [Github Issues](https://github.com/awslabs/fhir-works-on-aws-deployment/issues) for any feedback.
+We'd love to hear from you! Please reach out to our team via [GitHub Issues](https://github.com/awslabs/fhir-works-on-aws-deployment/issues) to provide any feedback.
```

Comment:
Suggested: "We would love to hear from you! Contact our team via [GitHub Issues] to provide any feedback."